pgn-mode.el

```diff
@@ -271,23 +271,23 @@
   "Return non-nil iff `pgn-mode--python-process' is running."
   (and pgn-mode--python-process (process-live-p pgn-mode--python-process) t))
 
-;; TODO: check for pyhon-chess
 ;; TODO: generalize script
 ;; TODO: pipes?
 (defun pgn-mode--make-process (&optional force)
   "Initialize pgn-mode `pgn-mode--python-process', optionally FORCE recreation if already exists."
+  (pgn-mode-python-chess-guard)
   (when (and (not force) (pgn-mode--process-running-p))
     (error "The pgn-mode Python process already running. Use optional `force' to recreate"))
-  (when (not (= 0 (call-process pgn-mode-python-path nil nil nil "-c" "import chess")))
```

Comment:
Why is this removed?

Reply:
It's replaced with `(pgn-mode-python-chess-guard)`.